docs/functions.rst

```diff
@@ -59,43 +62,46 @@ however functions which support this alternate API include the line:
 HIPBLAS_V2 and Deprecations
 ===========================
 
-As of hipBLAS version 2.0.0, hipblasDatatype_t is deprecated, along with all functions which use this type. In a future release, all uses of hipblasDatatype_t
-will be replaced by hipDataType. See the hipblasGemmEx documentation for a small exception where hipblasComputeType_t replaces hipblasDatatype_t for the
-computeType parameter.
+As of hipBLAS version 2.0.0, ``hipblasDatatype_t`` is deprecated, along with all functions which use this type. In a future release, all uses of ``hipblasDatatype_t``
+will be replaced by ``hipDataType``. See the :any:`hipblasGemmEx` documentation for a small exception where ``hipblasComputeType_t`` replaces ``hipblasDatatype_t`` for the
+``computeType`` parameter.
 
-hipblasComplex and hipblasDoubleComplex are also deprecated. In a future release, all uses of these types will be replaced with their HIP counterparts:
-hipComplex and hipDoubleComplex.
+``hipblasComplex`` and ``hipblasDoubleComplex`` are also deprecated. In a future release, all uses of these types will be replaced with their HIP counterparts:
+``hipComplex`` and ``hipDoubleComplex``.
 
-While hipblasDatatype_t, ``hipblasComplex``, and ``hipblasDoubleComplex`` are deprecated, users may use the compiler define or inline #define HIPBLAS_V2 before including the header file <hipblas.h> to
-access the updated API. In a future release, this define will no longer be needed and deprecated functions will be removed, leaving the updated interface.
+While ``hipblasDatatype_t``, ``hipblasComplex``, and ``hipblasDoubleComplex`` are deprecated, users may use the compiler define or inline ``#define HIPBLAS_V2`` before including the header file `<hipblas.h>` to access the updated API. In a future release, this define will no longer be needed and deprecated functions will be removed, leaving the updated interface.
 Please see the documentation for the following functions to see the new interfaces using hipDataType: hipblasTrsmEx, hipblasGemmEx,
```

Comment:
Sorry missed this, on L73 you might want the double `` around hipDataType, and maybe the function names.

Reply:
Fixed.